react-notes/src/layouts/MainLayout/components/Sidebar/Sidebar.js

```diff
@@ -4,17 +4,24 @@ import { Link } from 'react-router-dom';
 import styles from './Sidebar.module.css';
 import { ArticleLink } from './components';
 import { ArticlesContext } from '../../../../pages/article/context';
+import classNames from 'classnames';
```

Comment:
In case you don't have anything else to fix/refactor, I'd suggest playing around eslint (namely imports sorting here)

Remember that we are using CRA (create react app) here - this may affect how to install/configue eslint

Reply:
You may need to take a look at https://www.npmjs.com/package/@craco/craco

Reply:
Remember - this is absolutely optional for you, this is just something that is used in real-world projects and it may be just fun to play with.

If you want to take a rest of frontend development, you may slowly start to implement backend. I'd suggest NodeJS or Python for this, because knowing how backend works in this languages it will be easier to switch to C#. But it's again up to you :)

Reply:
Let me know what you've decided and feel free to resolve this thread